src/ProjectSchema/ModalSchemaPropertList.tsx

```diff
@@ -122,16 +121,16 @@ export function ModalSchemaProperties({
         </div>
         <p className={css.fieldSubText}>{schema.description || ""}</p>
       </header>
-      {(isFieldEditMode || !!hasContentsWithCurrentSchema) && (
+      {isFieldEditMode && (
         <>
           <FieldInput
-            type={currentEditProperty.type || "text"}
+            type={currentEditProperty.options.type || "text"}
             isEditMode={isFieldEditMode}
             inputValue={currentEditProperty.name}
-            changeInputHandler={handleChangePropertyName}
-            clickTypeHandler={() => setIsClickTypeIcon(!isClickTypeIcon)}
-            editHandler={handleClickEdit}
-            addHandler={handleClickAdd}
+            onFieldChanged={handleChangePropertyName}
+            onTypeClicked={() => setIsClickTypeIcon(!isClickTypeIcon)}
+            onEditClicked={handleClickEdit}
+            onAddClicked={handleClickAdd}
```

Comment:
```suggestion
            onAddClick={handleAddClick}
```

어제 props와 handler 명명에 대한 이야기를 하긴 했는데, 제가 전달을 잘못 드린 것 같습니다.. prop명으로는 on~ prefix를 사용하고 handler는 handle~ prefix를 사용, prefix + 명사 + 동사 혹은 prefix + 동사의 형태로 가는게 어떨까요..? 아무래도 리액트 기본 문법과 맞추는게 좋아보여서요!

Reply:
좋습니당~